src/main/java/net/quickwrite/noplayernotifier/utils/Permission.java

```diff
@@ -1,15 +1,24 @@
 package net.quickwrite.noplayernotifier.utils;
 
 /**
+ * The permissions that the plugin is checking.
+ *
  * @author QuickWrite
  */
 public enum Permission {
+    /**
+     * The permission for the {@code reload} command.
+     */
     RELOAD,
+
+    /**
+     * The permission so that no message will be send.
+     */
     BYPASS;
 
     /**
      * Returns a String variant of the permission
-     * with a <code>noplayernotifier.</code> in front
+     * with a {@code noplayernotifier.} in front
      *
      * @return The whole permission
      */
```

Comment:
The **a** is unnecessary.

```suggestion
    /**
     * Returns a String variant of the permission
     * with {@code noplayernotifier.} in front
     *
     * @return The whole permission
     */
```

Reply:
It is now fixed.